stubs/tests/ResourceTest.php

```diff
@@ -57,6 +57,29 @@ public function test_users_can_edit_resources_belonging_to_them()
         $response->assertRedirect(localized_route('resources.show', $resource));
     }
 
+    public function test_users_can_translate_resources_belonging_to_them()
+    {
+        if (! config('hearth.resources.enabled')) {
+            return $this->markTestSkipped('Resource support is not enabled.');
+        }
+
+        $user = User::factory()->create();
+        $resource = Resource::factory()->create(['user_id' => $user->id]);
+
+        $resource->setTranslation('title', 'en', 'title in English');
+        $resource->setTranslation('title', 'fr', 'title in French');
+
+        $this->assertEquals($resource->getTranslation('title', 'en'), 'title in English');
+        $this->assertEquals($resource->getTranslation('title', 'fr'), 'title in French');
+
+        $translationObject = ['en' => 'title in English', 'fr' => 'title in French'];
+
+        $this->assertEquals($resource->getTranslations('title'), $translationObject);
```

Comment:
The parameter order for `assertEquals` should be switched to match the method signature: https://phpunit.readthedocs.io/en/9.5/assertions.html#assertequals